java/squeek/veganoption/integration/waila/ProviderJutePlant.java

```diff
@@ -34,9 +35,9 @@ public ItemStack getWailaStack(IWailaDataAccessor accessor, IWailaConfigHandler
 		{
 			float growthValue = ((BlockJutePlant) accessor.getBlock()).getGrowthPercent(accessor.getWorld(), accessor.getPosition(), accessor.getBlockState()) * 100.0F;
 			if (growthValue < 100)
-				toolTip.add(String.format("%s : %.0f %%", I18n.format("hud.msg.growth"), growthValue));
+				toolTip.add(LangHelper.translate("waila.growth", (int) growthValue));
```

Comment:
I thought so as well. I'm guessing Mojang or Forge messed up the functionality at some point because it was very much not rounded when %.0f was used in the lang file :P